### ISSUE TITLE
Hyperparamter Optimisation 

### ISSUE THREAD

**Issue by benjaminjellis** [benjaminjellis/APPLE#1]
Use talos to perform hyperparameter optimisation

**Comment by benjaminjellis** [comment]
https://blog.tensorflow.org/2020/01/hyperparameter-tuning-with-keras-tuner.html

**Comment by benjaminjellis** [comment]
Currently using the dev branch to do this with talos. Early tests look like there ins't a huge gain to be made. 
Some options:

1. Continue with talos in the hops of fine tuning further 
2. Focus on more control during training, specifically on slowing down the learning rate or using call backs to end training early to prevent overfitting. More research needed on this.

**Comment by benjaminjellis** [comment]
Once the season is finished, reselect features and look into custom training.

**Comment by benjaminjellis** [comment]
Closing - shift of phase 2 priorities
